### PR TITLE
Update dead links for LiteSpeed

### PR DESCRIPTION
**Pull request by istiak101**
Updated old dead links to new links.

### CODE REVIEW

File: install/unix/litespeed.xml [hunk 1]
```diff
@@ -38,9 +38,9 @@
    <para>
     To obtain and install either LiteSpeed Web Server or OpenLiteSpeed Web Server, visit the
     LiteSpeed Web Server wiki
```
Comment (cmb69) [inline install/unix/litespeed.xml:40]: Apparently, the new documentation is no longer a Wiki, so the wording should be adjusted; maybe just "LiteSpeed Web Server documentation" here.

File: install/unix/litespeed.xml [hunk 2]
```diff
@@ -215,7 +215,7 @@ PHP_LSAPI_MAX_REQUESTS=500 PHP_LSAPI_CHILDREN=35 /path/to/lsphp -b IP_address:po
   OpenLiteSpeed Web Server, and Apache mod_lsapi. For steps on 
   server-side configuration, visit the wiki pages for 
   <link xlink:href="https://www.litespeedtech.com/support/wiki/doku.php/litespeed_wiki:php:configuring-lsws-for-php">LiteSpeed Web Server</link>
```
Comment (cmb69) [inline install/unix/litespeed.xml:217]: It seems this link should be updated as well.

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you for the PR!

File: install/unix/litespeed.xml (hunk 1 above)
Comment (istiak101) [reply install/unix/litespeed.xml:40]: I am updating the phrase.

File: install/unix/litespeed.xml (hunk 2 above)
Comment (istiak101) [reply install/unix/litespeed.xml:217]: I couldn't find a better link from current documentation. Let me compare again and see if that's possible.

**Pull request merged**